pocs/tests/test_pocs.py

```diff
@@ -220,32 +238,26 @@ def start_pocs():
         pocs.run(run_once=True, exit_when_done=True)
         assert pocs.is_weather_safe() is True
         pocs.power_down()
+        observatory.logger.info('start_pocs EXIT')
 
     pub = PanMessaging.create_publisher(6500)
     sub = PanMessaging.create_subscriber(6511)
 
-    pocs_process = Process(target=start_pocs)
-    pocs_process.start()
+    pocs_thread = threading.Thread(target=start_pocs)
+    pocs_thread.start()
 
-    # Wait for the running message
-    while True:
-        msg_type, msg_obj = sub.receive_message()
-        if msg_obj is None:
-            continue
+    # Wait for the RUNNING message,
+    wait_for_message(sub, attr='message', value='RUNNING')
 
-        if msg_obj.get('message', '') == 'RUNNING':
-            time.sleep(2)
-            # Insert a dummy weather record to break wait
-            observatory.db.insert_current('weather', {'safe': True})
+    time.sleep(2)
+    # Insert a dummy weather record to break wait
+    observatory.db.insert_current('weather', {'safe': True})
 
-        if msg_type == 'STATUS':
-            current_state = msg_obj.get('state', {})
-            if current_state == 'pointing':
-                pub.send_message('POCS-CMD', 'shutdown')
-                break
+    wait_for_message(sub, type='STATUS', attr='state', value='pointing')
```

Comment:
Note that we could probably change this from `pointing` to `scheduling` or `ready` to speed up the test slightly. Once it is out of `sleeping` the test is a success and the scheduling can potentially take non-trivial amounts of time.